FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Common/Extensions/ConfigurationRegistrationExtensions.cs

```diff
@@ -78,9 +89,64 @@ private static void ValidateConfiguration(this IServiceCollection services)
                 .GetRequiredService<IOptions<SchemaConfiguration>>()
                 .Value;
 
-            if (schemaConfiguration.EnableCustomizedSchema && string.IsNullOrEmpty(schemaConfiguration.CustomizedSchemaImageReference))
+            if (string.IsNullOrWhiteSpace(schemaConfiguration.SchemaImageReference))
+            {
+                if (schemaConfiguration.EnableCustomizedSchema)
+                {
+                    throw new ConfigurationErrorException($"Customized schema image reference can not be empty when customized schema is enable.");
+                }
+            }
+            else
+            { 
+                ValidateImageReference(schemaConfiguration.SchemaImageReference);
```

Comment:
Should we validate the image reference here if customized schema disabled?

Reply:
That is invalid config, shouldn't appear in our pipeline.

Reply:
But the configuration is provided by customer, so we can't ensure this invalid config will not appear? Is it better to skip image reference validation if customized schema is disabled?